scripts/deploy/main.bicep

```diff
@@ -637,7 +637,7 @@ resource messageContainer 'Microsoft.DocumentDB/databaseAccounts/sqlDatabases/co
       }
       partitionKey: {
         paths: [
-          '/id'
+          '/chatId'
```

Comment:
need: what do you think about renaming "chatId" to "chatSessionId"? the current naming leads to some confusion, giving the impression that there's a "chats" table